w3-blob.md

```diff
@@ -783,6 +783,104 @@ The `out.ok.size` MUST be set to the number of bytes that were freed from the sp
 
 Receipt MUST not have any effects.
 
+## Get Blob
+
+Authorized agent MAY invoke `/space/content/get/blob` capability on the [space] subject (`sub` field) to get Blobs added to it at the time of invocation.
+
+This may be used to check for inclusion, or to get the `size` property for a shard.
+
+### Get Blob Invocation Example
+
+Shown Invocation example illustrates Alice getting a blob stored on their space.
+
+```js
+{
```

Comment:
`// "/": "bafy..` is the CID of the object that is the invocation, it is referenced in the receipt. We use these comments so that the receipt (further down) can reference the invocation.

```suggestion

> ℹ️ We use `// "/": "bafy..` comments to denote CID of the parent object.

```js
{ // "/": "bafy..get"
```